sbi/inference/base.py

```diff
@@ -21,7 +22,7 @@ def __init__(
         self,
         simulator: Callable,
         prior,
-        x_o: Tensor,
+        x_shape: Optional[torch.Size] = None,
```

Comment:
should it be possible to pass a numpy shape?

Reply:
I was thinking about that too. But it's really so easy for the user to just make it a `torch.Size()`. And if they do not provide any (i.e. `None`) it's inferred anyway.

Reply:
We will make a type 
`Shape = Union[torch.Size, Tuple[int, ...]`